### PR TITLE
Rent & User models test

### PR DESCRIPTION
**Pull request by GabrielPiccini**
## Summary

The test for User and Rent models were implemented correctly.

- [ ] Test for User model.
- [ ] Test for Rent model.

## Screenshots

image

## TRELLO Card

https://trello.com/c/EbGUjjbe/42-crear-tests-para-el-modelo-de-rent-y-user

### CODE REVIEW

File: spec/models/rent_spec.rb [hunk 1]
```diff
@@ -1,5 +1,66 @@
...
+      is_expected.to be_valid
+    end
+  end
+
+  describe '#create' do
+    context 'When the user is nil' do
+      subject(:rent) do
+        build(:rent, user: nil)
+      end
+      it do
```
Comment (melissaregnier) [inline spec/models/rent_spec.rb:36]: add a newline between blocks

File: spec/models/rent_spec.rb [hunk 2]
```diff
@@ -1,5 +1,66 @@
...
+        is_expected.to be_invalid
+      end
+    end
+
+    context 'When the book is nil' do
+      subject(:rent) do
+        build(:rent, book: nil)
+      end
+      it do
```
Comment (melissaregnier) [inline spec/models/rent_spec.rb:46]: same here

File: spec/models/rent_spec.rb [hunk 3]
```diff
@@ -1,5 +1,66 @@
...
+        is_expected.to be_invalid
+      end
+    end
+
+    context 'When from is nil' do
+      subject(:rent) do
+        build(:rent, from: nil)
+      end
+      it do
```
Comment (melissaregnier) [inline spec/models/rent_spec.rb:56]: same here

File: spec/models/rent_spec.rb [hunk 4]
```diff
@@ -1,5 +1,66 @@
...
+        is_expected.to be_invalid
+      end
+    end
+
+    context 'When to is nil' do
+      subject(:rent) do
+        build(:rent, to: nil)
+      end
+      it do
```
Comment (melissaregnier) [inline spec/models/rent_spec.rb:66]: same here

File: spec/models/user_spec.rb [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+      is_expected.to be_valid
+    end
+  end
+
+  describe '#create' do
+    context 'When the first_name is nil' do
+      subject(:user) do
+        build(:user, first_name: nil)
+      end
+      it do
```
Comment (melissaregnier) [inline spec/models/user_spec.rb:40]: same here and all the others in this file

**Review by melissaregnier — CHANGES_REQUESTED** [review]

**Review by NicoFunes — APPROVED** [review]

**Review by melissaregnier — APPROVED** [review]

**Pull request merged**